ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightApiResult.kt

```diff
@@ -0,0 +1,34 @@
+/**
+ * Represents the result from the Shopper Insight API.
+ *
+ * @property eligible_methods Contains the payment methods available to the shopper.
+ */
+data class ShopperInsightApiResult (
+    val eligible_methods: PaymentMethods
```

Comment:
do we want to use @SerializedName and keep the property name camel case?

Reply:
Good catch, fixed!